core/src/main/scala/org/http4s/parser/LocationHeader.scala

```diff
@@ -12,7 +12,7 @@ trait LocationHeader {
 
   private class LocationParser(value: String) extends Http4sHeaderParser[Location](value) with Rfc3986Parser {
 
-    override def charset = StandardCharsets.ISO_8859_1
+//    override def charset = StandardCharsets.ISO_8859_1
```

Comment:
Part of a larger discussion about when and where to use `Charset`.  I had deleted the abstract member this was overriding, but I left the comment so I wouldn't forget `ISO-8859_1`.